FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,37 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let self = self else { return }
+			switch result {
+			case .failure:
+				completion(.failure(Error.connectivity))
+			case .success((let data, let response)):
+
+				guard response.statusCode == 200 else {
+					completion(.failure(Error.invalidData))
+					return
+				}
+
+				guard let feedLoaderItems = try? JSONDecoder().decode(FeedItemAPI.self, from: data).items else {
+					completion(.failure(Error.invalidData))
+					return
+				}
```

Comment:
Suggestion:
These two guard statements can be combined into one to remove duplication of `completion(.failure(Error.invalidData))`

```suggestion
				guard response.statusCode == 200, let feedLoaderItems = try? JSONDecoder().decode(FeedItemAPI.self, from: data).items else {
					completion(.failure(Error.invalidData))
					return
				}
```